src/Data/FingerTree/Strict.hs

```diff
@@ -0,0 +1,278 @@
+{-# LANGUAGE BangPatterns               #-}
+{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE FlexibleInstances          #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses      #-}
+{-# LANGUAGE StandaloneDeriving         #-}
+
+-- | Strict variants of 'FingerTree' operations.
+--
+-- TODO: Move this to @cardano-prelude@ as a new module, @Data.FingerTree.Strict@.
```

Comment:
This TODO can go now :slightly_smiling_face:

Reply:
```suggestion
```